server/__init__.py

```diff
@@ -313,9 +313,6 @@ def before_request():
     requested_locale = request.args.get('hl', i18n.DEFAULT_LOCALE)
     g.locale_choices = i18n.locale_choices(requested_locale)
     g.locale = g.locale_choices[0]
-    # Add commonly used config flags.
-    g.env_name = app.config.get('ENV_NAME', None)
```

Comment:
i think this was a much cleaner way to access the value (since it's used often)

Reply:
Revert this back. The intention was to know exactly that this is a field set from the Config, especially there is a mixer of using "g." and "current_app.conifg" in many files at the same time. But we can revisit consolidation them later.